ci/pipeline.yml

```diff
@@ -21,6 +21,9 @@ jobs:
 
       - task: unit-tests
         file: bosh-cpi-src/ci/tasks/unit-tests.yml
+        config:
+          params:
+            GO_VERSION: 1.8.1
 
   - name: build-candidate
```

Comment:
could this step verify the version of golang in the bosh release matches the container version? 

I like the spirit of this change but I believe this asserts the container runs a known version specified as an env variable. It doesn't verify the CPI is compiled with that version.